Samples/iOS-Swift/iOS-Swift-UITests/UserFeedbackUITests.swift

```diff
@@ -0,0 +1,178 @@
+//swiftlint:disable todo
```

Comment:
`l`: You can now remove the disable todo, as there is not todo anymore in this file 😃.
```suggestion
```

Reply:
oops, i already did and forgot to push the commit!